src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/package.json

```diff
@@ -3,9 +3,15 @@
   "displayName": "Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension",
   "description": "A companion extnesion for debugging Blazor WebAssembly applications in VS Code. Must be installed alongside the C# extension.",
   "version": "0.0.1",
+  "repository": {
+    "type" : "git",
+    "url" : "https://github.com/dotnet/aspnetcore-tooling.git"
+  },
+  "license": "Apache-2.0",
```

Comment:
Kathleen and Immo reviewed the last package.json bits that were published. We may need to have a similar review for our package.json to get the official stamp of approval for all the things here

Reply:
Pinged @danroth27 for review here and on the other item. He has the most context into the area.

@danroth27 Let me know if other PMs need to look at this.